util/configv3/load_config.go

```diff
@@ -128,6 +128,7 @@ func LoadConfig(flags ...FlagOverride) (*Config, error) {
 		CFTrace:          os.Getenv("CF_TRACE"),
 		CFUsername:       os.Getenv("CF_USERNAME"),
 		DockerPassword:   os.Getenv("CF_DOCKER_PASSWORD"),
+		CNBCredentials:   os.Getenv("CNB_REGISTRY_CREDS"),
```

Comment:
**question (non-blocking)**: To keep all CF-related environment variables namespaced, I prefer `CF_` prefixed, e.g., `CF_CNB_REGISTRY_CREDS`. Do you think this is still viable since the main was not released yet?

Reply:
I don't see an issue with renaming that. I'll check with the team.